open-source-trumps-innersource.md

```diff
@@ -0,0 +1,42 @@
+## Title
+Open Source trumps InnerSource
+* ORIGINAL: The Invented Here bias or internally developed
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+* ORIGINAL: Developers tend to select open source components rather than internally developed components, which results in poorer quality components or also makes it difficult to sustain internal component development.
+* NEW: Different BLs are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.
```

Comment:
Might this be a problem for a dedicated pattern/donut? I feel this is not so much related to OSS trumping InnerSource.

Reply:
You meant that this one could go to it's own pattern instead, right?
> Different Business Lines (BLs) are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.

Will leave this thread open but I think it should not prevent us from merging this PR.